src/bouwdossiers/batch.py

```diff
@@ -90,7 +90,101 @@ def openbaar_to_access(openbaar):
     return models.ACCESS_RESTRICTED
 
 
-def add_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+    """
+    Add wabo dossier to the the bouwdossier model. Structure of import is
+    almost identical to pre_wabo xml to avoid confusion since only a few
+    mappings are different and new fields are added.
+    """
+    dossier = x_dossier.get('intern_nummer')
+    stadsdeel, dossiernr = dossier.split('_')
+    titel = x_dossier.get('dossier_titel')
+
+    if not titel:
+        titel = ''
+        log.warning(f"Missing titel for Wabo dossier {dossiernr} in {file_path}")
+
+    datering = x_dossier.get('begindatum')
+    dossier_type = x_dossier.get('omschrijving')
+
+    bouwdossier = models.BouwDossier(
+        importfile=import_file,
+        dossiernr=dossiernr,
+        stadsdeel=stadsdeel.upper(),  # Make it upper to follow the same format as pre_wabo dossiers
```

Comment:
Maybe it's a good idea to do a check on whether the stadsdeel code which is provided here is correct and part of the possibilities/choices.